design/ceph/ceph-cluster-cleanup.md

```diff
@@ -7,8 +7,11 @@ As a rook user, I want to clean up data on the hosts after I intentionally unins
 ## Background
 
 ### Host-based cluster deletion
+If the user deletes a host-based cluster (running on raw devices and not on PVs) and starts a new cluster on the same hosts, then the user needs to perform the following manual steps:
```

Comment:
How about mentioning the following document instead of listing processes?

https://github.com/rook/rook/blob/master/Documentation/ceph-teardown.md#delete-the-data-on-hosts

Reply:
added this doc for reference. Still keeping the one liner for each step in this doc. Let me know what you think.

Reply:
It looks nice, thanks.